README.md

```diff
@@ -1,49 +1,87 @@
-# netbox-operator
+# NetBox Operator
 
 **Disclaimer:** This project is currently under development and may change rapidly, including breaking changes. Use with caution in production environments.
 
 NetBox Operator extends the Kubernetes API by allowing users to manage NetBox resources – such as IP addresses and prefixes – directly through Kubernetes. This integration brings Kubernetes-native features like reconciliation, ensuring that network configurations are maintained automatically, thereby improving both efficiency and reliability.
 
-<div align=center>
-    <img src="./docs/NetBox Operator High-Level Architecture.png" alt="Diagram: NetBox Operator High-Level Architecture" width="800"/>
-    <p><em>Figure 1: NetBox Operator High-Level Architecture</em></p>
-</div>
+![Figure 1: NetBox Operator High-Level Architecture](docs/netbox-operator-high-level-architecture.drawio.svg)
 
 # Getting Started
 
 ## Prerequisites
+
 - go version v1.22.0+
-- docker version 17.03+.
-- kubectl version v1.11.3+.
-- Access to a Kubernetes v1.11.3+ cluster.
+- docker version 17.03+
+- kubectl version v1.11.3+
+- Access to a Kubernetes v1.11.3+ cluster
+- kind
+- docker cli
```

Comment:
Is this not included with docker by default? Does no harm to keep though

Reply:
Let's keep it since there may be users that want to use another runtime such as podman.